internal/templates/internal/add_member.html

```diff
@@ -0,0 +1,29 @@
+{% extends "internal/base.html" %}
+{% load i18n %}
+
+{% block body %}
+    <div class="ui container">
+        <h1>{% trans "Add new member" %}</h1>
+        <form class="ui form" method="POST" action="">
+            {% csrf_token %}
+            <div class="ui two fields">
+                <div class="ui field">
+                    <label>{{ form.user.label }}</label>
+                    {{ form.user }}
+                </div>
+                <div class="ui field">
+                    <label>{{ form.date_joined.label }}</label>
+                    {{ form.date_joined }}
+                </div>
+            </div>
+            <div class="ui field">
+                <label>{{ form.committeess.label }}</label>
```

Comment:
An `s` too much. The label is not shown.

Reply:
Do not know how that slipped through. Have fixed it nonetheless.